.gitignore

```diff
@@ -1,5 +1,6 @@
-
 .DS_Store
 test.html
-jsdoc/
+
 node_modules/
+dist/
```

Comment:
Thank you for your suggestions.
As of now, /dist is for providing the pre-build SDK & sample of config.js.
To make the implementation (not build) process easier for anybody including non-techy, we need an alternative way to provide these files.

Reply:
I think that `ingestly.js` may be provided from CDN like `cdnjs`. But, `page_code.js` is not so. This file is placed in folder like `example/` with `index.js` that includes this module...